File: csharp/delete_calendar.cs
var client = new RestClient("https://graph.microsoft.com/v1.0/me/calendar");
var request = new RestRequest(Method.DELETE);
request.AddHeader("content-type", "application/json");
request.AddParameter("application/json", "{ }", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);

File: python/delete_calendar.py
import http.client

conn = http.client.HTTPSConnection("graph.microsoft.com")

payload = "{ }"

headers = { 'content-type': "application/json" }

conn.request("DELETE", "/v1.0/me/calendar", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
